packages/docs/docs/contracts-api/verifiers/epoch-key-verifier-helper.md

```diff
@@ -78,7 +129,7 @@ function verifyAndCheck(
 Verify an [epoch key proof](../../circuits-api/circuits#epoch-key-proof) and validate the public signals against the onchain state. This function will revert if any inputs are invalid. This is identical to `verifyAndCheck` but also checks that the caller is the attester.
 
 :::caution
-This function does not require the epoch for the proof to be the current epoch. The user may generate a valid proof for a past epoch. If you require the proof to be for the current epoch you should add an additional check using [`attesterCurrentEpoch`](#attestercurrentepoch).
+This function **does not** require the epoch for the proof to be the **current epoch**. The user may generate a valid proof for a past epoch. If you require the proof to be for the current epoch you should add an additional check using [`attesterCurrentEpoch`](#attestercurrentepoch).
```

Comment:
make 'does not' and 'current epoch' bold font on line 110 the same as here
--same for 2 CAUTION boxes in `epoch-key-lite-verifier-helper.md` and `reputation-verifier-helper.md`